unbalanced_dataset/base.py

```diff
@@ -181,3 +182,25 @@ def fit_sample(self, X, y):
         """
 
         return self.fit(X, y).sample(X, y)
+
+
+    def transform(self, X):
```

Comment:
For consistency, I think that with the current API a resampler should never implement a transform method. Transform method in the pipeline object will be required only if is implemented by the last estimator.